signals/settings.py

```diff
@@ -22,7 +22,7 @@ def load_settings(settings_path):
     return setting_dict
 
 
-def save_settings(paths, schema, generator_name, data_models, core_data, project_name):
+def save_settings(paths, templates, schema, generator_name, data_models, core_data, project_name):
```

Comment:
we're passing `templates` into these two methods now but not storing or loading them